Nested Folders: Update documentation (#81054)
grafana/grafana: docs/sources/dashboards/manage-dashboards/index.md

As of Grafana 10.2, there is no longer a special **General** folder. Dashboards without a folder are now shown at the top level alongside folders.
{{% /admonition %}}

> ### Shared with me
> 
> The 'Shared with me' section displays folders and dashboards that are directly shared with a user which would otherwise not be visible in the standard dashboards tree. This is usually because the user doesn't have access to one or more of the parent folders.
> 

Reviewer: Use bold for UI text (https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold).

```suggestion
### Shared with me

The **Shared with me** section displays folders and dashboards that are directly shared with a user which would otherwise not be visible in the standard dashboards tree.
This is usually because the user doesn't have access to one or more of the parent folders.

```

As of Grafana 10.2, there is no longer a special **General** folder. Dashboards without a folder are now shown at the top level alongside folders.
{{% /admonition %}}

### Shared with me

> The **Shared with me** section displays folders and dashboards that are directly shared with a user which would otherwise not be visible in the standard dashboards tree.

Reviewer 2: Changing this to second person vs third

```suggestion
The **Shared with me** section displays folders and dashboards that are directly shared with you that wouldn't otherwise be visible in the standard dashboards tree.
```

Reviewer 2: Also, is there another word we can use instead of "tree"--it's just sort of been introduced here without any previous definition of what we mean. Does the word "list" suffice?

> This is usually because the user doesn't have access to one or more of the parent folders.

Reviewer 2: ```suggestion
This is usually because you don't have access to one or more of the parent folders.
```

Reviewer 2: Is there another use case that we should be mentioning? If it's not a relevant use case, can we just say, "This is because" and take out the word "usually"?

1. On the **Dashboards** page, click **New** and select **New folder** in the drop-down.
1. Enter a unique name and click **Create**.

> When you save a dashboard, you can optionally select a folder for the dashboard to be saved in.
> 
> **You can also create a folder within an existing folder:**
> 
> 1. Click **Dashboards** in the main menu.
> 1. On the **Dashboards** page, click an existing folder name to navigate to that folder.
> 1. On the folder's page, click **New** and select **New folder** in the drop-down.
> 1. Enter a unique name and click **Create**.
> 
> You can nest folders up to 4 levels deep.

Reviewer: This suggestion mainly just avoids passive voice.
```suggestion
When you save a dashboard, you can optionally select a folder to save the dashboard in.

You can also create a folder within an existing folder:

1. Click **Dashboards** in the main menu.
1. On the **Dashboards** page, click an existing folder name to navigate to that folder.
1. On the folder's page, click **New** and select **New folder** in the drop-down.
1. Enter a unique name and click **Create**.

You can nest folders up to four levels deep.
```

1. On the **Dashboards** page, click **New** and select **New folder** in the drop-down.
1. Enter a unique name and click **Create**.

When you save a dashboard, you can optionally select a folder to save the dashboard in.

You can also create a folder within an existing folder:

1. Click **Dashboards** in the main menu.
1. On the **Dashboards** page, click an existing folder name to navigate to that folder.
1. On the folder's page, click **New** and select **New folder** in the drop-down.
1. Enter a unique name and click **Create**.

> You can nest folders up to four levels deep.

Reviewer 2: Can this just be an optional step within the create a dashboard folder task? Kind of like:

1. Click Dashboards in the main menu.
2. Do one of the following:
  - On the Dashboards page, click New and select New folder in the drop-down.
  - Click an existing folder and on the folder’s page, click New and select New folder in the drop-down.
 3. Enter a unique name and click Create.

When you nest folders, you can do so up to four levels deep.
